src/pages/Script/ScriptsComp.tsx

```diff
@@ -0,0 +1,275 @@
+import { useState, useEffect, ReactNode, Dispatch, SetStateAction, useMemo } from 'react'
```

Comment:
There is too much `any` in this file, most of which could be improved, and you should try to avoid using `any` to break the TypeScript type system.

Reply:
> There is too much `any` in this file, most of which could be improved, and you should try to avoid using `any` to break the TypeScript type system.

I reconstituted all `any` with creating interfaces and types